server/action_creators.js

```diff
@@ -80,6 +80,17 @@ export function addEntriesToOrder(orderId, newEntries) {
   };
 }
 
+export function setReadyForBill(orderId) {
+  return (dispatch, getState) => {
+    return Order.findOneAndUpdate({ id: orderId }, { status: constants.READY_FOR_BILL })
+    .then(() => {
+      return Order.getOrders();
```

Comment:
I thought about this last week and concluded that because the database is the source of truth, it would make sense to fetch all the update the state with the new order. Would there be a case where we incrementally update the state, and but there is a discrepancy between the database and the state? Or do you think it's okay because we get the result of the updated record, and just incrementally update the state with the updated record.